src/client/routes/profile/user.graphql.ts

```diff
@@ -0,0 +1,31 @@
+import { gql } from 'apollo-boost';
+
+export default gql`
```

Comment:
I'm a bit confused on how we get types from this @leonm1

Reply:
Great point! I'll attempt to explain:

The types are extracted mainly from `src/common/schema.graphql.ts` (we can break this up if we need) by `graphql-code-generator`. In addition to extracting types from this file, the code generator will look through all of `src/client` recursively looking for `*.graphql.ts` files (could've used regular graphql files but maybe someone wants to PR that?). Those discovered files are all the queries and mutations for which hooks will be generated.